### PR TITLE
Add development board information

### PR DESCRIPTION
**Pull request by jamescfkung**
- Add development board suggestion information for community user
- Add Mediatek MT8173 link

### CODE REVIEW

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jforissier** [comment]
`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`

File: README.md [hunk 1]
```diff
@@ -68,9 +69,13 @@ please read the file [build_system.md](documentation/build_system.md).
 | [STMicroelectronics b2020-h416](http://www.st.com/web/catalog/mmc/FM131/SC999/SS1633/PF253155?sc=internet/imag_video/product/253155.jsp)|`PLATFORM=stm-orly2`|
 | [Allwinner A80 Board](http://www.allwinnertech.com/en/clq/processora/A80.html)|`PLATFORM=sunxi`|
 | [HiKey Board (HiSilicon Kirin 620)](https://www.96boards.org/products/hikey/)|`PLATFORM=hikey`|
-| MediaTek MT8173 EVB Board|`PLATFORM=mediatek-mt8173`|
+| [MediaTek MT8173 EVB Board](http://www.mediatek.com/en/products/mobile-communications/tablet/mt8173/)|`PLATFORM=mediatek-mt8173`|
 | Texas Instruments DRA7xx|`PLATFORM=ti-dra7xx`|
 
+### 3.1 Development board for community user
+For community user, we suggest that [Hikey board](https://www.96boards.org/products/ce/hikey/) as your developemnt board. It provides detail hardware document including chip datasheet, board schematic, ...etc. and also related open source softwares download link on the website.
```
Comment (jenswi-linaro) [inline README.md]: Please break this line at 79 columns or something.

s/community user/community users/

s/we suggest that/we suggest using/

s/as your/as/

s/detail hardware document including chip datasheet, board schematic/detailed hardware documentation including chip datasheet, board schematics/

**Comment by jamescfkung** [comment]
Updated according to Jens' comments

File: README.md [hunk 2]
```diff
@@ -68,9 +69,15 @@ please read the file [build_system.md](documentation/build_system.md).
 | [STMicroelectronics b2020-h416](http://www.st.com/web/catalog/mmc/FM131/SC999/SS1633/PF253155?sc=internet/imag_video/product/253155.jsp)|`PLATFORM=stm-orly2`|
 | [Allwinner A80 Board](http://www.allwinnertech.com/en/clq/processora/A80.html)|`PLATFORM=sunxi`|
 | [HiKey Board (HiSilicon Kirin 620)](https://www.96boards.org/products/hikey/)|`PLATFORM=hikey`|
-| MediaTek MT8173 EVB Board|`PLATFORM=mediatek-mt8173`|
+| [MediaTek MT8173 EVB Board](http://www.mediatek.com/en/products/mobile-communications/tablet/mt8173/)|`PLATFORM=mediatek-mt8173`|
 | Texas Instruments DRA7xx|`PLATFORM=ti-dra7xx`|
 
+### 3.1 Development board for community user
+For community users, we suggest using [Hikey board](https://www.96boards.org/products/ce/hikey/)
+as developemnt board. It provides detailed documentation including chip
```
Comment (pascal-brand-st-dev) [inline README.md]: s/developemnt/development

File: README.md [hunk 3]
```diff
@@ -68,9 +69,15 @@ please read the file [build_system.md](documentation/build_system.md).
...
+datasheet, board schematics, ...etc. and also related open source softwares
```
Comment (pascal-brand-st-dev) [inline README.md]: s/softwares/software

**Comment by jenswi-linaro** [comment]
With Pascals comments addressed
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by jamescfkung** [comment]
Content updated and add "Review-By" in commit message

**Comment by pascal-brand-st-dev** [comment]
The signed-off is missing

**Comment by jamescfkung** [comment]
Add signed-off

**Comment by jamescfkung** [comment]
Rebased and push again

**Pull request merged**